dbeam/src/main/scala/com/spotify/dbeam/JdbcAvroIO.java

```diff
@@ -55,7 +55,9 @@
 import org.slf4j.LoggerFactory;
 
 public class JdbcAvroIO {
+
   private static final String DEFAULT_CODEC = "deflate";
+  public static final CodecFactory DEFAULT_DEFLATE_CODEC = CodecFactory.deflateCodec(6);
```

Comment:
Ah, again partial confusion on my last hurried review. I understood this number 6 had some other meaning than a fluffy level. I think now this is better as I understand it now as a opinionated level measure on the compressions. Probably cannot make it clearer without explaining where the number came from :)
But good enough! 👍